site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -98,14 +98,35 @@ to a TAS ResourceFlavor.
 
 {{< include "examples/tas/sample-job-preferred.yaml" "yaml" >}}
 
+### ClusterAutoscaler support
+
+TAS integrates with the [Kubernetes ClusterAutoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
+through the [Provisioning AdmissionCheck](/docs/admission-check-controllers/provisioning/).
+
+When a workload is assigned to the TAS Resource Flavor with Provisioning
+AdmissionCheck, then its admission flow has the following stages:
+1. **Quota reservation**: quota is reserved, and the workload obtains the
```

Comment:
```suggestion
1. **Quota reservation**: quota is reserved, and the Workload obtains the
```

In this description, it looks like `workload` indicates `Workload` object instead of generic term.